### PR TITLE
Docker Keycloak setup

### PR DESCRIPTION
**Pull request by latch2112**

### CODE REVIEW

File: Keycloak/docker-compose.yml [hunk 1]
```diff
@@ -29,13 +29,17 @@ services:
...
+    volumes:
+      - .:/tmp
     environment:
       KEYCLOAK_USER: "admin"
       KEYCLOAK_PASSWORD: ${KEYCLOAK_PASSWORD}
       DB_VENDOR: "postgres"
       DB_PASSWORD: ${POSTGRESQL_PASSWORD}
       JDBC_PARAMS: "connectTimeout=30"
       DB_ADDR: "postgres"
+      KEYCLOAK_IMPORT: /tmp/realm-export.json
```
Comment (nyanofthemoon) [inline Keycloak/docker-compose.yml:41]: Where is realm-export.json ?

Comment (latch2112) [reply Keycloak/docker-compose.yml:41]: damn

**Review by nyanofthemoon — APPROVED** [review]
Approved once it's tested and working with `clin-proxy-api` Auth Service :)

**Comment by latch2112** [comment]
Working with ```clin-proxy-api```

**Pull request merged**